README.md

```diff
@@ -82,12 +82,12 @@ This is the main Component, build around a `ScrollView` component, that should w
 |containerStyle| [View Style](https://reactnative.dev/docs/view-style-props) | Container Styles passed to the `ScrollView` |
 |keyboardHiddenContainerStyle| [View Style](https://reactnative.dev/docs/view-style-props) | Container Styles passed to the `ScrollView`. Applied only when the keyboard is hidden |
 |scrollViewProps| `Object` - [ScrollView Props](https://reactnative.dev/docs/scrollview#props) | Props passed to the `ScrollView`|
-|safeMarginContentHeight| `Number` | Safe Margin to determine if content fits between top of screen and keyboard. |
-|safeMarginBottom| `Number` | Safe Margin between slected input or form and keyboard |
+|safeMarginContentHeight| `Number` | Safe Margin to determine if content fits between top of screen and keyboard. `contentHeight` is the height of the elements that are withing the `<Avoidable>` or `<Avoidable.Area>` tags. You might want to fine-tune this so by changing the `safeMarginContentHeight` you can add or subtract from it. Default `0` |
+|safeMarginBottom| `Number` | `safeMarginBottom` is the distance between the keyboard and the desired element that it gets aligned to. If you want to leave some space between the keyboard and your input or form change this. Default `0` |
 
 ### `Avoidable.Area`
 
-This is the component to specify where your form or selection of inputs would be. This is only used with `alignTo = 'bottom'` and is neccesary when `contextAware = true`.
+Only when `contextAware = true` and `alignTo = 'bottom'` must an Area be defined in order for the package to work properly. This is the component to specify where your form or selection of inputs is.
```

Comment:
"must an Area be defined" - Is it just me, or it sounds a big like Yoda? :D 
"an `Area` must be defined" sounds better to me. What do you think?

Reply:
"This is the component to specify where your form or selection of inputs is." -> "This component specifies the components/inputs that should avoid the keyboard"?